docs/manuals/long-jobs.qmd

```diff
@@ -0,0 +1,68 @@
+# Running long jobs on your Research Cloud workspace
+
+## When is this relevant?
+
+Often it is advisable to run long jobs (e.g. long scripts or analyses that take hours or days to complete) as background processes. 
+In the following cases long running jobs will be interrupted when you close the window with which you access your workspace:
+
+- When you access your workspace via your web browser (Jupyter Notebooks, RStudio, Ubuntu Desktop, etc.)
+- When you access your workspace via SSH
+
+[More info on access methods](https://utrechtuniversity.github.io/vre-docs/docs/first-steps.html#getting-access-to-your-workspace)
+
+## How to run jobs as background processes
+
+### Prerequisites
+- You will need some basic linux command line skills to be able to run scripts as background processes. If you don't have these skills, take some time to practice using sections 1, 2, 3 and 7 of this short [online course](https://swcarpentry.github.io/shell-novice) before proceeding. You can practice in the terminal (see step 1).
+- Your script should be 'standalone'; which means that you should be able to run your entire script in one go without modifications. All figures and output data should be stored in files (e.g. `.png` images and/or `.csv` tables, this is not strictly necessary for jupyter notebooks).
```

Comment:
> without modifications

Maybe: "without providing additional input" (or: interactive input)

> this is not strictly necessary for jupyter notebooks

I found this a little bit confusing; why not?